externals/kyuubi-spark-sql-engine/src/test/scala/org/apache/kyuubi/engine/spark/schema/RowSetSuite.scala

```diff
@@ -237,7 +237,7 @@ class RowSetSuite extends KyuubiFunSuite {
     assert(r6.get(9).getStringVal.getValue === "2018-11-06")
 
     val r7 = iter.next().getColVals
-    assert(r7.get(10).getStringVal.getValue === "2018-11-17 13:33:33.600")
+    assert(r7.get(10).getStringVal.getValue === "2018-11-17 13:33:33.6")
```

Comment:
why change precision?

Reply:
> why change precision?

https://github.com/apache/incubator-kyuubi/blob/6fa1327f4e58ca79e7c07f8e089d6ca312b323c9/externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/schema/RowSet.scala#L222-L223

The length of fraction is from 0 to 9, precision not changed, just the following '0' is not displayed.

Reply:
We shall keep the padding

Reply:
increased the minWidth